rts/Lua/LuaHandle.cpp

```diff
@@ -1614,6 +1615,8 @@ void CLuaHandle::Update()
 
 	// call the routine
 	RunCallIn(L, cmdStr, 0, 0);
+
+	LuaVFSDownload::ProcessDownloads();
```

Comment:
you can make LuaVFSDownload an eventClient and then change ProcessDownloads() to Update()